README.md

```diff
@@ -43,11 +44,21 @@ This component handles the background jobs for key generation and transaction si
 - Joins the key generation session using the provided session ID.
 - Stores the key generation data securely upon completion.
 
-
 ### Transaction Signing
 
 #### Job: Transaction Signing
 
 - Triggered when a transaction signing request is made.
 - Joins the transaction signing session using the provided session ID.
 - Waits for all parties to join and complete the signing process.
+
+## How to setup vultisigner to run locally?
+
+- Setup redis using docker
```

Comment:
can you please provided detail , how to start redis using docker, we have docker compose file